app/models/subnet_domain.rb

```diff
@@ -2,9 +2,6 @@ class SubnetDomain < ActiveRecord::Base
   belongs_to :domain
   belongs_to :subnet
 
-  validates :subnet_id, :presence => true
-  validates :domain_id, :presence => true, :uniqueness => {:scope => :subnet_id}
```

Comment:
I'm not sure, where would an api user will get an error if the domain /
subnet combination is not unique? I dont want to get a FK error in an api
call.